.github/workflows/ci-backend.yml

```diff
@@ -491,8 +491,6 @@ jobs:
               uses: actions/setup-python@v5
               with:
                   python-version: ${{ matrix.python-version }}
-                  cache: pip
-                  cache-dependency-path: '**/requirements*.txt'
```

Comment:
Looks like `uv sync` manages its own installs and doesn’t really populate `~/.cache/pip`, so pip caching here probably won’t speed things up.